### DISCUSSION TITLE
Shopee ads search result filter

### DISCUSSION THREAD

**Discussion by prasetyodedy** [realodix/AdBlockID#1726] [Ideas]
mohon tambahkan sebaris filter iklan di hasil pencarian shopee ini ke AdBlockID
`shopee.*##.shopee-search-item-result__item:has([data-sqe="ad"])`

dan juga sebaris filter iklan rekomendasi toko pada hasil pencarian
`shopee.*##.shopee-header-section--simple.shopee-header-section`

**Comment by realodix** [comment]
Saya butuh beberapa link untuk mengkonfirmasinya. Tanpa itu, Saya tidak dpt menambahkannya.

**Reply by prasetyodedy** [reply]
https://www.reddit.com/r/indonesia/comments/12fd588/comment/jffddgc/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button


tangkapan layar sebelum menggunakan filter
![Screenshot_20240315_205716_Kiwi Browser](https://github.com/realodix/AdBlockID/assets/88732071/f9f263b0-0645-4526-93d6-c8d565d3b771)



tangkapan layar setelah ditambahkan filter
`shopee.*##.shopee-search-item-result__item:has([data-sqe="ad"])`
menghilangkan iklan di baris pencarian.
tapi masih ada iklan rekomendasi toko
![Screenshot_20240315_205826_Kiwi Browser](https://github.com/realodix/AdBlockID/assets/88732071/357b6f76-7c93-47be-93b8-c04489d8a1e0)



tangkapan layar setelah ditambahkan filter iklan rekomendasi toko
`shopee.*##.shopee-header-section--simple.shopee-header-section`
![Screenshot_20240315_205940_Kiwi Browser](https://github.com/realodix/AdBlockID/assets/88732071/b076fee0-6286-4186-94c1-5b330b726777)

**Comment by realodix** [comment]
> bro, kok ngga ada respond, udah dikasih bukti juga

Halo Bro @prasetyodedy, sorry banget Saya kebanjiran notifikasi, jadinya terlewat.

Filter ini bisa gak? `shopee.co.id##:matches-path(/search?) .shopee-search-item-result > .shopee-search-item-result__items:has(a div[data-sqe="ad"])`

Saat ini Kami sedang membahasnya di https://github.com/realodix/AdBlockID/issues/1732

Saya mengalami kendala dimana halamannya dibutuhkan login, mungkin bro bisa bantu Saya dengan memberikan file zip halamannya, agar Saya bisa melihat struktur kodenya.

![318061410-25ca898f-3a6c-4b6a-841c-502ed95916f1](https://github.com/realodix/AdBlockID/assets/1314456/817f50bc-577c-4dfc-aaea-de9d702bd664)

**Reply by prasetyodedy** [reply]
[Shopee_search_result.zip](https://github.com/realodix/AdBlockID/files/14814117/Shopee_search_result.zip)
done bro

**Reply by realodix** [reply]
Ini pas Saya buka, isinya hanya 1 file, pas Saya ubah ke html, isinya benar-benar berantakan. Biasanya setelah Anda save file html seperti gambar di atas, akan ada 1 file html dan 1 folder, makanya Saya minta zip.

**Reply by prasetyodedy** [reply]
maaf notifnya kelewat,
sebenernya bukankah tinggal nambah baris yang sudah saya kasih tahu taruh di filterlist AdBlockID.
toh itu juga hasil selection by uBlock Origin mas

**Comment by realodix** [comment]
Selesai, ada di AdBlockID Plus ya.

**Reply by prasetyodedy** [reply]
mantap mas, thank you.
jadi ngga perlu nambah manual tiap pindah device/browser.
work prfectly

**Comment by raffized12-web** [comment]
Hallo mas izin bertanya ada kah yg tau codingan untuk nambah auto ads balance di sller center shopee makasih sebelim nya
